testsuite/domain/src/test/resources/host-configs/host-minimal.xml

```diff
@@ -59,7 +59,7 @@
 
     <domain-controller>
         <!-- Remote domain controller configuration with a host and port -->
-        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm">
+        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false">
```

Comment:
My comment here is more of a note for probably post CR1 work.

This file is used in ProductInfoUnitTestCase and ServerManagementTestCase.

For ProductInfoUnitTestCase it shouldn't matter which setting the file has, so changing it to match what other tests need is fine. 

ServerManagementTestCase is pretty messy. It has tests related to server lifecycle. Ideally those would be tested with "ignoring" set both ways, as a big part of the functionality here involves the HC having or obtaining the necessary config data as servers are added. The class also has tests related to profile cloning, and profile and socket-binding-group includes. One of these, testCloneProfile() just won't work if the slave is ignoring, but it doesn't really belong in this class anyway, as server management is not a factor in the test. It should be moved out, or perhaps have the slave assert changed if the slave is ignoring. For some of the others, server management *is* relevant, but they would need to be reworked to properly test things if the slave is ignoring. Basically, check that "config present" != ignoring, and then add the server config and check that "config present" == true. Where "config" is the profile/s-b-g/server-group in question.